pyproject.toml

```diff
@@ -139,50 +140,56 @@ psutil = "*"
 timezonefinder = "*"  # just used for nav ETA
 pycryptodome = "*"  # used in updated/casync, panda, body, and a test
 
+[tool.poetry.group.docs.dependencies]
+Jinja2 = "*"
+sphinx = "*"
+sphinx-rtd-theme = "*"
+sphinx-sitemap = "*"
+
+[tool.poetry.group.testing.dependencies]
+coverage = "*"
+hypothesis = "~6.47"
+mypy = "*"
+pre-commit = "*"
+pytest = "*"
+pytest-cov = "*"
+pytest-cpp = "*"
+pytest-subtests = "*"
+pytest-xdist = "*"
+pytest-timeout = "*"
+pytest-randomly = "*"
+pytest-asyncio = "*"
+pytest-mock = "*"
+pytest-repeat = "*"
+ruff = "*"
+
+[tool.poetry.group.tools.dependencies]
```

Comment:
let's only put zstd under this one for now. I agree that these go through, but the xx pipeline shouldn't have metadrive installed, so we'll have to think some more about how to split these up